tests/functional/api/test_index.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+# String type for request/response headers and metadata in WSGI.
+#
+# Per PEP-3333, this is intentionally `str` under both Python 2 and 3, even
+# though it has different meanings.
+#
+# See https://www.python.org/dev/peps/pep-3333/#a-note-on-string-types
+native_str = str
+
+
+@pytest.mark.functional
+class TestGetIndex(object):
+    def test_it_returns_links(self, app):
+        res = app.get("/api/")
+
+        assert "links" in res.json
+        assert res.status_code == 200
+
+    def test_it_returns_links_for_resources(self, app):
```

Comment:
Note that the tests in this module test for the presence of expected entries but do not assert that extra entries don't exist. This is in large part because the structure of the returned data object is a bit funky, with sub-resources getting keys at the same level as the actions on their parent resources. I'm thinking of proposing that we change the structure of this response in v2, which means that v1 shouldn't really change going forward, anyway.